code/datums/diseases/advance/symptoms/rodman.dm

```diff
@@ -0,0 +1,178 @@
+					if(prob(5))
+						to_chat(M, "<span class='notice'>[pick(
+						"Your thoughts seem to be clearing up.",
+						"Your brain feels great.",
+						"The fog in your mind is clearing up.")]</span>")
+					return
+				if((increase_volume == TRUE) && (prob(35)))
+					M.say( pick( list(
+					"Ook!!", 
+					"Ook.", 
+					"Ooh-ooh!!",
+					"Ah-ah!!",
+					"Eek-eek!!",
+					"Hoo hoo!!", 
+					"SCREE!!") )) //More volume means shouted messages now
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 100, 1)
+					return
+				if(prob(20))
+					M.say( pick( list(
+					"Ook!",
+					"Ook.",
+					"Eek", 
+					"Oop?", 
+					"Aak-eek.", 
+					"Chee."
+					) )) //Return to monke.
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 50, 1)
+			if((!ishuman(M)) && (awaken_monkey == TRUE) && (is_monkey == TRUE)) //Confirm they were a monkey from the start, currently a monkey and that the threshold is hit.
+				awaken_monkey = FALSE // Only one attempt at this, to minimize spam.
+				if((!M.mind) && (isliving(M))) //Confirm they are mindless and alive
+					//Give them a name that isn't just Monkey(420)
+					M.name = pick( list (
+					"Marcel", 
+					"King Louie", 
+					"Chim Chim", 
+					"Caesar", 
+					"Betsy", 
+					"Joe", 
+					"Cornelius", 
+					"Monty", 
+					"Dodger", 
+					"Bonzo", 
+					"Bozo", 
+					"Clements", 
+					"Stephen Flinger", 
+					"Mincing Monkey", 
+					"Normal Guy", 
+					"Petey", 
+					"Babbels", 
+					"Funky Monkey", 
+					"Kilgor Monkeyman",
+					"Monkey McFly",
+					"Chunky Monkey",
+					"Monkey-anda Jones",
+					"Diddy",
+					"Thelonious Monkey",
+					"Alexander the Monkey",
+					"Tarzan", "Mokeysha",
+					"Potassium Phil",
+					"ExcAPE",
+					"OranguTim",
+					"Ook Operative",
+					"Doctor Zaius",
+					"Greg",
+					"Kurtchak",
+					"Rolls",
+					"Flips",
+					"Bitey",
+					"Curious Geoff",
+					"Ookie Won Camonkie",
+					"Anaookin Bananawalker",
+					"Dr. Banana Pockets",
+					"Ook",
+					"Dave",
+					"Sherlok Monke",
+					"Monkie Holms",
+					"Freddy Krooker",
+					"Tim the Sorcerous",
+					"Aperham Linkook",
+					"Apevar",
+					"Ook'sie",
+					"Returned",
+					"Sopa de Macaco",
+					"Macho Monkey Randy Savook",
+					"Mr.Bubbles"
+					) )

```

Comment:
You might want to put all of this in a txt file and use world.file2list to get all of the names